docs/openj9_support.md

```diff
@@ -72,26 +71,93 @@ show the minimum build configurations that we have tested to build an OpenJDK bi
 **NOTE:** If you obtain pre-built binaries from [AdoptOpenJDK.net](https://adoptopenjdk.net/index.html),
 platform support might vary, depending on their build environment.
 
-### OpenJDK 8 build environment
+### OpenJDK 8
+
+OpenJDK8 binaries are supported on the minimum operating system levels shown in the following tables:
+
+
+| Linux                                 |   x32  |  x64   | ppc64le    | Z31  | Z64 |
+|---------------------------------------|--------|--------|------------|------|------|
+| Centos 6                              |   Y    |   Y    |     Y      |   N  |  N   |
+| Centos 7                              |   Y    |   Y    |     Y      |   N  |  N   |
+| Red Hat Enterprise Linux (RHEL) 6     |   Y    |   Y    |     Y      |   Y  |  Y   |
+| RHEL 7                                |   Y    |   Y    |     Y      |   Y  |  Y   |
+| SUSE Linux Enterprise Server (SLES) 12|   Y    |   Y    |     Y      |   Y  |  Y   |
+| Ubuntu 16.04                          |   Y    |   Y    |     Y      |   N  |  Y   |
+| Ubuntu 18.04                          |   Y    |   Y    |     Y      |   N  |  Y   |
+
+<i class="fa fa-pencil-square-o" aria-hidden="true"></i><span class="sr-only">Note</span> **Note:** Not all of these distributions are tested, but Linux distributions that have a
```

Comment:
This shows Note twice, with the second occurrence in bold.

Reply:
Above OpenJDK 8 there is text which should be removed "The following tables show the minimum build configurations that we have tested to build an OpenJDK binary."

Reply:
Thanks for reviewing. I should get in the habit of using WIP in the title. 

The "Note twice" is only an issue when viewed in GitHub, not on the staging site. GitHub does not understand the HTML <span> so repeats the word.

Removed the text from the top paragraph as you pointed out.